loading-kernel-module/vm-create

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Is this necessary here?  It seems like it duplicates https://docs.fedoraproject.org/en-US/fedora-coreos/provisioning-libvirt/ ?